ceph-grafana-trigger/config/definitions/ceph-grafana-trigger.yml

```diff
@@ -23,7 +23,7 @@
       - string:
           name: CONTAINER_VERSION
           description: "The version tag for the containers; will have -ARCH added to it.  By convention, the version of Grafana"
-          default: "6.7.4"
```

Comment:
Just curious: any specific reason for 8.2.6 and not 8.4.2 or 8.1.2?

Reply:
I was checking this out - https://hub.docker.com/r/grafana/grafana and the latest stable version i found here is 8.2.6, so decided to go forward with this.

Reply:
Makes sense!